### ISSUE TITLE
[MPT-30] index: style banners

### ISSUE THREAD

**Issue by wei2912** [AdvisorySG/mentorship-page#604]
The homepage banners (e.g. programme logo, partners, description text) should be styled nicely to match the aesthetic of [https://beta.advisory.sg/](https://beta.advisory.sg/) and for clarity.

This task involves some element of design which should be discussed before starting.

<sub>From [SyncLinear.com](https://synclinear.com) | [MPT-30](https://linear.app/advisorysg/issue/MPT-30/index-style-banners)</sub>

**Comment by wei2912** [comment]
Please note these details in your design:
1. Have two banners on the website, one for the introductory text ("The Advisory Mentorship Programme pairs students...") and one for the "What can you expect?".
2. Banners should be restricted in width, occupy two rows and have similar font styling to the the post carousel on https://beta.advisory.sg/. (If this requires changing font family, do standardise the entire website.)
3. Privacy text should be shifted to the bottom (creation of a proper footer will be in a separate issue).

I've left notes on our dicussions on https://beta.advisory.sg at https://github.com/AdvisorySG/dawn-advisory-theme/issues/175.
